server/src/main/java/org/candlepin/audit/EventSinkImpl.java

```diff
@@ -209,10 +224,12 @@ public void sendEvents() {
 
     @Override
     public void rollback() {
-        log.warn("Rolling back ActiveMQ transaction.");
+        log.warn("Rolling back ActiveMQ transaction");
+
         try {
-            ClientSession session = getClientSession();
+            ClientSession session = this.getClientSession();
             session.rollback();
+            session.close();
```

Comment:
Put the close() into a finally block so that an attempt is always made to close the session, even on rollback fails. If closing the session fails, there's not really anything we can do -- log and move on.